Common/NPCs/ConditionalDropping/ConditionalDropHandler.cs

```diff
@@ -9,29 +10,42 @@ internal class ConditionalDropHandler : GlobalNPC
 {
 	public override void ModifyNPCLoot(NPC npc, NPCLoot npcLoot)
 	{
-		if (npc.type is NPCID.GoblinArcher or NPCID.GoblinPeon or NPCID.GoblinScout or NPCID.GoblinSorcerer or NPCID.GoblinThief or NPCID.GoblinWarrior)
+		// Handle goblin NPCs
+		if (npc.type is NPCID.GoblinArcher or NPCID.GoblinPeon or NPCID.GoblinScout or
+			NPCID.GoblinSorcerer or NPCID.GoblinThief or NPCID.GoblinWarrior)
 		{
-			AddCountCondition(npcLoot, LocalizedText.Empty, ModContent.ItemType<TomeOfTheElders>(), 8);
+			AddCountCondition(npcLoot, "Tome Drop", ModContent.ItemType<TomeOfTheElders>(), 8);
```

Comment:
I wonder if some sort of Consts for this to keep track of these better.
Or utilize the Class.Name attribute or something for this instead

Reply:
Not a requirement